src/vs/platform/actions/common/menuService.ts

```diff
@@ -442,19 +442,16 @@ function createMenuHide(menu: MenuId, command: ICommandAction | ISubmenuItem, st
 	};
 }
 
-function createMenuKeybindingAction(menu: MenuId, command: ICommandAction | ISubmenuItem, when: ContextKeyExpression | undefined = undefined, commandService: ICommandService, keybindingService: IKeybindingService): IAction | undefined {
-	if (isISubmenuItem(command)) {
-		return undefined;
-	}
-
-	const configureKeybindingAction = toAction({
-		id: `configureKeybinding/${menu.id}/${command.id}`,
-		label: keybindingService.lookupKeybinding(command.id) ? localize('change keybinding', "Change Keybinding") : localize('configure keybinding', "Configure Keybinding"),
+export function createConfigureKeybindingAction(commandId: string, when: ContextKeyExpression | undefined = undefined, commandService: ICommandService, keybindingService: IKeybindingService): IAction {
+	const hasKeybinding = !!keybindingService.lookupKeybinding(commandId);
+	return toAction({
+		id: `configureKeybinding/${commandId}`,
+		label: hasKeybinding ? localize('change keybinding', "Change Keybinding") : localize('configure keybinding', "Configure Keybinding"),
```

Comment:
I think in the other cases this is called "Change Keybinding"

![image](https://github.com/microsoft/vscode/assets/900690/4b595017-4413-4778-b496-68073851d3fb)